utilities/scripts/xmlPagsToEntityPags.groovy

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
It's not really bootstrap or items to support the build, though after you suggested an ant task to run it as opposed to just invoking using groovy directly I can see arguments for saying it is there to support the build.  I can also see arguments for putting it under uportal-war/src/utilities or similar since it is uportal-specific and not global.  

I guess now that the build invokes it as a target, putting it under bootstrap is fine though that still bothers me as somewhat incongruent with the items in there (right now only ant-related library jar files).